python/pylibcudf/pylibcudf/libcudf/detail/__init__.pxd

```diff
@@ -0,0 +1 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
```

Comment:
All the copyrights probably need updated to reflect SPDX changes in https://github.com/rapidsai/cudf/pull/20321. Can you check on that? Maybe merge the upstream and run `pre-commit run --all-files`.